src/main/java/com/fakesitewoopasdf/songr/Routes.java

```diff
@@ -16,6 +16,8 @@
 
```

Comment:
the naming conventions of some of your files are kind of off, best practices encourage you to name your controller files to have the word controller. for example this one could be `AlbumController.java` and then you would want to have a separate one for other routes.